hydra-node/test/Hydra/Chain/Direct/Contract/Mutation.hs

```diff
@@ -245,6 +246,8 @@ propTransactionIsNotObserved (tx, _) st =
   case observeSomeTx tx st of
     Nothing ->
       property True
+    Just (OnInitTx{}, _) ->
+      property True
```

Comment:
Don't do this here. We should rather question the expectation at the call site of `propTransactionIsNotObserved`